pkg/aws/validations/iam_helpers.go

```diff
@@ -0,0 +1,60 @@
+package validations
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/service/iam"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/openshift/rosa/pkg/aws/tags"
+	semver "github.com/hashicorp/go-version"
+)
+
+func GetRoleName(prefix string, role string) string {
+	name := fmt.Sprintf("%s-%s-Role", prefix, role)
+	if len(name) > 64 {
```

Comment:
Can we save 64 as a const and export it and update it in ROSA, looks like we have it hard coded in places.